workflow-service/src/main/java/com/redhat/parodos/workflow/definition/dto/WorkFlowPropertiesDefinitionDTO.java

```diff
@@ -10,7 +12,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@JsonDeserialize(builder = WorkFlowPropertiesDefinitionDTO.WorkFlowPropertiesDefinitionDTOBuilder.class)
+@JsonDeserialize(builder = WorkFlowPropertiesDefinitionDTOBuilder.class)
```

Comment:
@eloycoto I had to update here and add dependency on `workflow-service-sdk` to successfully generate javadocs. Please take a look.